test/native/MODCLUSTER-785/testit.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+source ../includes/script.bash
+
+# check IMG...
+if [ -z ${IMG} ]; then
+  echo "IMG needs to defined, please try"
+  echo "export IMG=quay.io/${USER}/tomcat_mod_cluster"
+  exit 1
+fi
+# check mod_proxy_cluster or mod_cluster
+if [ -z $USE13 ]; then
+  SOURCES=https://github.com/modcluster/mod_proxy_cluster
+  MDBRANCH=main  
+else
+  SOURCES=https://github.com/modcluster/mod_cluster
+  MDBRANCH=1.3.x  
+fi
+
+# first stop any previously running tests.
+stoptomcats
+removetomcats
+
+# and httpd
+docker stop MODCLUSTER-785
+docker rm MODCLUSTER-785
+
+# build httpd + mod_proxy_cluster
+#
+LOCREPO=`git remote -v | grep origin | grep fetch | awk ' { print $2 } ' | awk -F/ ' { print $4 } '`
+REPO="https://raw.githubusercontent.com/modcluster/mod_proxy_cluster/main/test/native/MODCLUSTER-785/mod_proxy_cluster.conf"
```

Comment:
I don't like those names. `LOCREPO` is more `ORIGINREPO` or `REPOORIGIN` or something like that, and `REPO` is not repository at all. (Maybe `MPCCONF` or `REMOTECONF`?).

Reply:
I am bad with names... I will fix that.